cluster/environments/azure-multiple-clusters/README.md

```diff
@@ -29,11 +36,18 @@ The Service Principal that is configured for authentication must have a Owner ro
 
 ### 2. Service Principals
 #### Authentication Service Principal
-Create a Azure service principal for authentication with Azure subscription with the [`Owner`](https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles#azure-rbac-roles) role in the subscription with the following [`az ad sp create-for-rbac`](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create) command:
+If one is wanting to deploy a cluster with a Service Principal having `Owner` level permissions, one must create a Azure service principal for authentication with Azure subscription with the [`Owner`](https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles#azure-rbac-roles) role in the subscription with the following [`az ad sp create-for-rbac`](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create) command:
```

Comment:
Consider deleting "If one is..." and "one must..." language.  Just say "If you want to deploy..." and "create a Azure.."